android/assets/jsons/Civ V - Vanilla/Nations.json

```diff
@@ -469,7 +469,7 @@
 		"outerColor": [225,105,0],
 		"innerColor": [255,255,78],
 		"uniqueName": "Wayfinding",
-		"uniques": ["Can embark and move over Coasts and Oceans immediately", "+1 Sight when embarked", "+[10]% Strength if within [2] tiles of a [Moai]"],
+		"uniques": ["Can embark and move over Coasts and Oceans immediately", "[+1] Sight when [Embarked]", "+[10]% Strength if within [2] tiles of a [Moai]"],
```

Comment:
Good catch, but wrong unique.
"+[1] Sight for all [Embarked] units - changed in base game.